go/kbcmf/packets.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package kbcmf
+
+import ()
+
+type receiverKeysPlaintext struct {
+	GroupID    int    `codec:"gid"`
+	MACKey     []byte `codec:"mac,omitempty"`
+	SessionKey []byte `codec:"sess"`
+}
+
+type receiverKeysCiphertext struct {
+	KID  []byte `codec:"kid"`
```

Comment:
"KID" is something of a Keybase-specific term. Maybe the right term here is "public" or "public_key"? Though since we have the group number inside the encrypted keys, does order matter? Maybe the receivers list should just turn into a map?

Speaking of which, do we need the entire recipient public key in here? I'm leaning towards "it's just cleaner to have the whole thing" but I wanted to think out loud.